Lenovo ThinkPad X13 Yoga Gen 1/README

```diff
@@ -0,0 +1,11 @@
+Manufacturer: Lenovo
+Model Name: ThinkPad X13 Yoga Gen 1
+Model Number:
+
+Notes: 
+
```

Comment:
Please add this line to the Notes section for a quick overview of the sensor:
~~~
USB AES 056A:5220 (4k, tilt)
~~~